src/Xhgui/Controller/Run.php

```diff
@@ -9,6 +9,23 @@ public function __construct($app, $profiles, $watches)
         $this->_watches = $watches;
     }
 
+    public function homepage()
+    {
+        // TODO: how to fetch DIC from app?
+        global $di;
```

Comment:
Agree, the "global" part was a quick&dirty solution just to get this to work (I already have my fork deployed in production). Can you point me to a proper way of doing this? I'm not familiar with Slim (only worked with Symfony/Silex and Zend Framework)